server/plugin/command.go

```diff
@@ -501,6 +518,163 @@ func (p *Plugin) handleIssue(_ *plugin.Context, args *model.CommandArgs, paramet
+						hook.Events = []string{"*"}
+						break
+					}
+					if err := p.CheckOptionsValid(strings.Split(val, ",")); err != nil {
+						return err.Error()
+					}
+					hook.Events = strings.Split(val, ",")
+				}
+			}
+		}
+		if _, exists := config["url"]; !exists {
+			config["url"] = siteURL
+		}
+		if _, exists := config["insecure_ssl"]; !exists {
+			config["insecure_ssl"] = false
+		}
+		if _, exists := config["content_type"]; !exists {
+			config["content_type"] = "x-www-form-urlencoded"
+		}
+		if len(hook.Events) == 0 {
+			hook.Events = []string{"*"}
+		}
+		hook.Config = config
+		githubHook, _, err := p.CreateHook(ctx, githubClient, owner, repo, hook)
+		if err != nil {
+			return err.Error()
+		}
+
+		hookDetails, _, err := p.GetHook(ctx, githubClient, owner, repo, *githubHook.ID)
+		if err != nil {
+			return err.Error()
+		}
+
+		hookURL := baseURL + owner
+		if repo != "" {
+			hookURL += "/" + repo
+		}
+		hookURL += "/settings/hooks/"
+		txt := "Webhook Created Successfully \n"
+		hookID := strconv.Itoa(int(*githubHook.ID))
+		txt += fmt.Sprintf(" *  [%s](%s) :  -  %s \n", hookID, hookURL+hookID, strings.Join(hookDetails.Events, " , "))
+		return txt
+	case list:
+
+		if len(parameters) != 1 {
+			return "Unknown action, please use `/github help` to see all actions available."
+		}
+
+		owner, repo := parseOwnerAndRepo(parameters[0], p.getBaseURL())
+		var githubHooks []*github.Hook
+		var githubResponse *github.Response
+		var err error
+		opt := &github.ListOptions{
+			PerPage: 10,
+		}
+		if repo == "" {
+			for {
+				githubHooks, githubResponse, err = githubClient.Organizations.ListHooks(ctx, owner, opt)
+				if err != nil {
+					return err.Error()
+				}
+				if githubResponse.NextPage == 0 {
+					break
+				}
+				opt.Page = githubResponse.NextPage
+			}
+		} else {
+			for {
+				githubHooks, githubResponse, err = githubClient.Repositories.ListHooks(ctx, owner, repo, opt)
+				if err != nil {
+					return err.Error()
+				}
+				if githubResponse.NextPage == 0 {
+					break
+				}
+				opt.Page = githubResponse.NextPage
+			}
+		}
+
+		var txt string
+		if len(githubHooks) == 0 {
+			txt = "Currently there are no webhook in this [Owner/Repo]"
+		} else {
+			txt = "### Webhook in this Repositories\n"
+		}
+
+		for _, hook := range githubHooks {
+			hookID := strconv.Itoa(int(*hook.ID))
+			hookURL := baseURL + owner
+			if repo != "" {
+				hookURL += "/" + repo
+			}
+			hookURL += "/settings/hooks/"
+			txt += fmt.Sprintf(" *  [%s](%s) :  -  %s \n", hookID, hookURL+hookID, strings.Join(hook.Events, " , "))
+		}
+
+		return txt
+	}
+	return "Invalid action, only `add` and `list` commands are supported."
+}
+func (p *Plugin) CheckOptionsValid(options []string) error {
+	for _, val := range options {
+		for _, item := range Manifest.Props["webhook_events"].([]interface{}) {
+			if item.(string) != val {
+				return errors.New(val + " is not a valid events to trigger webhook")
+			}
+		}
+	}
```

Comment:
What's the point of having these in the manifest versus in the code?

Reply:
Sorry , i misunderstood the comment from hanzei https://github.com/mattermost/mattermost-plugin-github/pull/456#discussion_r670259213 , fixed this in this commit.